.clang-tidy

```diff
@@ -26,9 +26,18 @@ Checks:     '
             performance-*,
             portability-*,
             readability-*,
+            -bugprone-exception-escape,
+            -bugprone-macro-parentheses,
```

Comment:
This one seems reasonable to reenable in the future, I just didn't want to tackle it right now.